### ISSUE TITLE
Team member not able to access tool

### ISSUE THREAD

**Issue by chaosbuster** [FIRST-Tech-Challenge/fmltc#245]
_**Spinning off a separate issue for a discussion that started within the [Java Null Direction Issue #238](https://github.com/FIRST-Tech-Challenge/fmltc/issues/238).  Thank you to Chris for sharing [the post](https://community.ftclive.org/t/granting-students-team-members-access/45) regarding granting team members access.  We are also aware of Section 4.5 in the [FTC Machine Learning Toolkit Manual](https://storage.googleapis.com/ftc-ml-firstinspires-prod/docs/ftc-ml_manual_2021.pdf).**_

I have three FTC Teams (Alpha 19541, Beta 19770 and Gamma 20771).  As Coach and Team Administrator on my FIRST Dashboard for these teams, I am able to access the FML Toolchain and select one of these teams before going into the tool.  I am also able to Add Team Members through the FTC-Scoring.   

Once I was able to access the FML tool, a team member on our Gamma team tried to access the tool and was not successful.   He is an Accepted team member with consent & release on my Dashboard in firstinspires.org.  My understanding is that his parent created a FIRST inspires account to complete this process and only provided an email for our student team member.

After this first attempt at gaining access, I ADDed the student as a Team Member within the FTC Scoring site using the same email that his parent provided in FIRST Inspires (per Section 4.5 in the Manual linked above and as [described in the Community Forum](https://community.ftclive.org/t/granting-students-team-members-access/45)).  The team member is still unable to access the tool.    He is also unable to create an account within FIRST Inspires since his email is already identified / used in the system.

In addition we have: 
- sent an email to firstteammembers@firstinspires.org
- we are attempting to see if a student on our Alpha team who indicates he has a FIRST Inspires account is able to access the FML Toolchain.

**Comment by texasdiaz** [comment]
Yes, student accounts through the FIRST Dashboard system can be a bit hairy because there are two ways to register students - and only one way is supportable through the FIRST Dashboard SSO to access ftc-ml. This is not a bug in ftc-ml, and is definitely not actionable through fmltc. I will leave this bug item open until it is documented in the manual.

**Comment by chaosbuster** [comment]
So should we have students use their parent's FIRST Dashboard system login for now?  Or use our own Team Administrator account to submit their videos and train the model for them under our account?

**Comment by texasdiaz** [comment]
I cannot ask your students to use their parent's accounts, as the students would need to know their parent's passwords (which is not recommended). 

I actually had the same (type of) issue with my daughter when I created an account for her when she was 12 and then she wanted her own account when she was 13. My daughter got around this by creating her own FIRST Dashboard account using a new Gmail account, then registered as a student. On the parent side it gets a little confusing since it looks like I have two kids with the same name, but I filled out the consent and release for both accounts and we registered her second account with her FTC team too (the coach "dismissed" her original account, so there weren't two of her on the team roster). I eventually called FIRST and they were able to remove the original youth account that I created for her, and the second account worked fine.
